Dockerfile.pipelines

```diff
@@ -129,6 +135,11 @@ RUN if [ $installWithPip == "true" ] ; then \
           make compiler-api ; \
           pip install -e . ; \
         popd ; \
+      elif [ $DEVITO_BACKEND == "ops" ] ; then \
+        git clone https://github.com/opesci/OPS.git /tmp/OPS ; \
```

Comment:
so...you clone under `/tmp`, but you build under `$HOME`. Did I get it right? Not sure why, but fine

Reply:
this seems different than what happens on Travis

Reply:
yes, indeed. it's a different path than from Travis
i just reproduced the pattern i've found on their respective scripts using yask as a template

in Travis yask is cloned into `$/HOME/build/opesci/yask` and in pipelines it is cloned into `/tmp/yask`